src/test/java/org/folio/mr/service/MediatedRequestActionsServiceTest.java

```diff
@@ -67,6 +72,12 @@ class MediatedRequestActionsServiceTest {
   @Mock
   private SearchService searchService;
 
+  @Mock
+  private SearchClient searchClient;
+
+  @Mock
+  private SystemUserScopedExecutionService executionService;
```

Comment:
Here we mock `SystemUserScopedExecutionService` but never define any behavior for it. It is being called when you run your tests, right? What does it return?